FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -527,9 +527,12 @@ protected function setAttributes()
         /** @var Product\Manufacturer $supplier */
         $supplier = $this->productStruct->getManufacturer();
         if ($supplier) {
-            $xmlSupplier = new Attribute('brand');
-            $xmlSupplier->setValues([StaticHelper::removeControlCharacters($supplier->getName())]);
-            $allAttributes[] = $xmlSupplier;
+            $supplierName = trim(StaticHelper::removeControlCharacters($supplier->getName()));
```

Comment:
Check if `\FinSearchUnified\Helper\StaticHelper::cleanString` could be used instead (since it calls `trim` and `\FinSearchUnified\Helper\StaticHelper::removeControlCharacters` anyway).

Reply:
@ihanli i guess we can call `\FinSearchUnified\Helper\StaticHelper::cleanString` as it removes potential bad characters as well.